packages/hdwallet-native/src/native.ts

```diff
@@ -82,15 +125,20 @@ export class NativeHDWallet
   _isNative = true;
 
   readonly #deviceId: string;
+  transport?: core.Transport;
```

Comment:
Don't need this anymore

Reply:
my IDE automatically added it I guess because it's technically part of the interface, but I'll remove it.